src_docs/source/test_results/sync_tests.rst

```diff
@@ -2,7 +2,7 @@
 
    <link rel="stylesheet" type="text/css" href="../_static/css/override-width.css">
 
-Automation Status
+Sync Tests
 =================
 
 All the visualizations are dynamic so feel free to play with them.
```

Comment:
![Screenshot from 2022-09-30 15-35-15](https://user-images.githubusercontent.com/97231749/193293777-913ba7bb-9899-46af-a19e-76a091ce3ca4.png)

The links have the same text, it would be nice to differentiate them to know what we are about to see.

Reply:
makes sense; thank you